### ISSUE TITLE
Getting an error on a phpbb forum no matter what settings I use

### ISSUE THREAD

**Issue by wadbr** [lairdshaw/fups#3]
The error I get is 

> In FUPSBase::scrape_forum_pg in line 1778 in file /home2/laird/public_html/apps/fups/classes/CFUPSBase.php: Error: couldn't find any topic matches on one of the forum pages. The URL of the page is https://forum.mobilism.org/viewforum.php?f=1&start=0

The url in there is a valid url with a few topics in it, no matter what I try (with https, no https, user topics, date selections, username etc) FUPS will not see any topics.

All the topics I'm trying to get are publically accessible. 

What might be the problem here? A different version of phpbb? 

The output file I get is this 
```


{
    "1": {
        "topics": []
    },
    "character_set": "UTF-8"
}
```

**Comment by lairdshaw** [comment]
It looks like that forum is using a custom skin. If you have an account on it, try logging in and changing your skin to one of the defaults (subsilver or prosilver). Then provide login credentials in your FUPS settings.

**Comment by wadbr** [comment]
I just tried that, changed to the classic theme but it failed at login (the login page is a custom theme when you are not signed in). This is my error code '844a5bbbf9a4d64341b067cc0b0e517a'. I logged out from the forum incase they have a one session per user but it's still not loggin in and showing me the same error.

**Comment by lairdshaw** [comment]
In that case, the only solution is to customise the FUPS code - potentially you could get away with simply adding a custom regular expression for 'login_success', but you might need to customise the check_do_login() method itself in classes/CphpBB.php too.

**Comment by wadbr** [comment]
Yeah, I would probably have to dig down into the code itself. Thank you for your help and amazing documentations though, this will be a good and fun learning experience because of it.
